xbmc/android/activity/XBMCApp.cpp

```diff
@@ -299,6 +301,38 @@ bool CXBMCApp::EnableWakeLock(bool on)
   return true;
 }
 
+bool CXBMCApp::EnableAudioFocus(bool on)
```

Comment:
In cases such as this I much prefer separate methods, i.e. `EnableAudioFocus()` and `DisableAudioFocus()`. The caller code becomes much more readable since you don't have to look up what the lonely boolean parameter does.